### ISSUE TITLE
Incompatible with Foundry v12

### ISSUE THREAD

**Issue by maxobremer** [ReAcTiOnN77/combat-exhaustion#4]
Was testing this module on foundry v12.331 and DnD5e v4.0.4 and this module doesnt seem to work.
weird thing, i tried to change the spell slots used on a player character and it kinda froze the game and made it not possible to roll an item etc
Now for the actual usefull stuff, here are the warning the console log gave
![image](https://github.com/user-attachments/assets/f7828714-f5fa-4179-9fc5-532dbfdbdd04)

Now it only mentions deprecation warnings since v12 but there are probably also some issues with DnD5e v4 as that overhauled plenty as wll

**Comment by thepting** [comment]
Yeah, I'm hitting this now too as I'm prepping to migrate my campaign to v12.  

It's causing REALLY bad performance issues.  Easy way to replicate the issue it to plop a token on the board, set HP to zero, then roll a death save.  At that point the FPS rate tanks, and dragging the character sheet around the screen is super sluggish.

**Comment by ReAcTiOnN77** [comment]
Sorry had a lot of life stuff going on I have now updated the module to be v13 compatible. However fix not implemented for v12 unfortunately.
